dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/monitor/DatadogRumMonitorTest.kt

```diff
@@ -830,7 +958,7 @@ internal class DatadogRumMonitorTest {
         val runnable = testedMonitor.keepAliveRunnable
 
         testedMonitor.handleEvent(mockEvent)
-        Thread.sleep(200)
```

Comment:
just a thought: maybe we can return `Future` type from `handleEvent` so that we can avoid `Thread.sleep` calls? (it makes tests flaky imo). it is `internal` method anyway, so we are free to modify it.